src/asm/rgbasm.1

```diff
@@ -105,8 +203,11 @@ $ rgbasm -o bar.o - < foo.asm
 The resulting object file is not yet a usable ROM image \(em it must first be
 run through
 .Xr rgblink 1
-and
+and then
 .Xr rgbfix 1 .
```

Comment:
Surely that’s implied?

Reply:
Thought it wouldn't hurt to explicit it. You know someone is going to try.